samples/excel/90-scenarios/currency-converter.yaml

```diff
@@ -0,0 +1,178 @@
+order: 5
+id: excel-scenarios-currency-converter
+name: Currency Converter
+description: 'Reads rows and update base currency column, based on value date.'
+author: cakriwut
+host: EXCEL
+api_set:
+    ExcelApi: '1.4'
+script:
+    content: |
+      /*
+       * Copyright (c) Riwut Libinuko. All rights reserved. Licensed under the MIT license.
+      */
+
+      declare let moment: any;
+      let tblSetup: Record<string, number> = {};
+
+      $("#convert").click(() => tryCatch(convert));
+      $("#setup").click(() => tryCatch(setupSample));      
+
+      /** Main converting function **/
+      async function convert() {
+        await Excel.run(async (context) => {
+          let range = context.workbook.getSelectedRange(); 
+          // Check if any cell inside table is selected.
+          let table = range.getTables();
+          table.load("items")    
+          await context.sync();
+          if(table.items.length == 0){
```

Comment:
Since we name the table in `setupSample`, we can use `TableCollection.getItem` to reference it. That way, we don't need all this error checking. We also don't need to have the user select the table.

Reply:
Removed